src/core/EventDispatcher.js

```diff
@@ -2,11 +2,9 @@
  * https://github.com/mrdoob/eventdispatcher.js/
  */
 
-function EventDispatcher() {}
+class EventDispatcher {
```

Comment:
`EventDispatcher` should not be part of this PR.

Reply:
> `EventDispatcher` should not be part of this PR. Better to put this change in your `core` PR.

Thank you review, Let me try.